### DISCUSSION TITLE
Enclosure/s

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#82] [Ideas]
Hi Lars,

firstly, I think you need to reposition the item circled in red in this image. I think it should be at the far right (followed by rearrangement of text related items).

![image](https://github.com/GrandOrgue/GoOdf/assets/6999990/b7b75f8c-4ed8-4d2e-9f89-284e0ae4b9e3)

Secondly, is there a way of selecting adjustments of the enclosure along the horizontal X-axis besides the current vertical Y-axis mouse movements?

**Comment by larspalo** [comment]
So, you'd like to have the TextBreakWidth spinctrl moved down to its own row at the bottom then?

For the second question I'm not entirely sure what you mean. Please clarify if needed. It's certainly possible to create the enclosure as a horizontal slider. The mouse axis values are for click/drag adjustments, which generally works somewhat poorly. It's much better to use the mouse wheel to adjust enclosure position up and down (or left/right as orientation really doesn't matter).
